tools/generate-helpers/pg-table-bindings/schema.go.tpl

```diff
@@ -72,9 +73,13 @@ var (
             }...)
             {{- end }}
         {{- end }}
+
         {{- if or (.Obj.IsGloballyScoped) (.Obj.IsDirectlyScoped) (.Obj.IsIndirectlyScoped) }}
             schema.ScopingResource = resources.{{.Type | storageToResource}}
         {{- end }}
+        {{- if .PermissionChecker }}
+            schema.PermissionChecker = {{ .PermissionChecker }}
+        {{- end }}
```

Comment:
I feel this should be mutually exclusive. In either, standard permission evaluation is done on `schema.ScopingResource` or a custom permission evaluator is used.

Reply:
This is actually mutually exclusive. When a `PermissionChecker` is present in the schema, the SAC checks rely on the permission checker to perform access control. Otherwise the `ScopingResource` is used.
I made the generation part more explicit there.